### ISSUE TITLE
security center contact put return 201 but swagger only defines status code 200 for 2017-08-01-preview

### ISSUE THREAD

**Issue by njuCZ** [Azure/azure-rest-api-specs#10717]
in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/securityContacts.json#L148, it only defines status code 200 is acceptable, but it seems the backend service has changed its behaviour and return status code 201.
Currently all released sdk are using this swagger, the go sdk found 201 is not included in the acceptable status code list and will return error.

I found https://github.com/Azure/azure-rest-api-specs/blob/master/specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json#L137 has fixed it, but there is no version in readme.md that reference this file.

we should generate new sdk that use the new version file

**Comment by tombuildsstuff** [comment]
@njuCZ is this a breaking behavioural change, since this used to work previously?

**Comment by njuCZ** [comment]
@tombuildsstuff yes, the backend service should not change the behaviour for old api-version.

**Comment by AddySanto** [comment]
chlahav doesn't own this now.  Looking for appropriate replacement contact.
